dev-python/uri_template/uri_template-1.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="URI Template expansion in strict adherence to RFC 6570"
+HOMEPAGE="https://gitlab.linss.com/open-source/uri_template/"
+SRC_URI="https://gitlab.linss.com/open-source/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
```

Comment:
Have you really tested it on all these architectures?

Reply:
I have not, but going by what I can test that would mean downgrading jsonschema to ~amd64 only.

Reply:
Yes, that's how we handle new packages.